Mixpanel/Mixpanel.m

```diff
@@ -1596,6 +1609,10 @@ - (void)showNotificationWithObject:(MPNotification *)notification
             if (shown && ![notification.title isEqualToString:@"$ignore"]) {
                 [self markNotificationShown:notification];
             }
+
+            if (shown == NO) {
```

Comment:
Should be `if (!shown)`.

See https://www.bignerdranch.com/blog/bools-sharp-corners/